src/convnet_export.js

```diff
@@ -1,7 +1,7 @@
 (function(lib) {
   "use strict";
   if (typeof module === "undefined" || typeof module.exports === "undefined") {
```

Comment:
This already takes care of seeing if you are in browser env or not. Node env has module/module.exports, otherwise you are loading the lib from browser, thus `window` will be defined.